task.js

```diff
@@ -28,7 +28,17 @@ class Task {
     this.mapper = mapper || function() { return ''; };
     this.translations = [];
   }
+  static async logError(title,error){
+    var today = new Date();
+    var date = today.getFullYear()+'-'+(today.getMonth()+1)+'-'+today.getDate();
+    var time = today.getHours() + ":" + today.getMinutes() + ":" + today.getSeconds();
+    var dateTime = date+' '+time;
+    const otfl = `${OUTPUT_DIRECTORY}/errors.txt`;
```

Comment:
Best to create an error file per mapper.  Have a look at `get publicOutputFile()` and `get privateOutputFile()` in this class and create a similar one `get errorFile()`.

Reply:
I get the error :+1: 
Cannot read property 'errorFile' of undefined
when I do the following